viewer-admin/src/main/java/nl/b3p/viewer/admin/stripes/ApplicationActionBean.java

```diff
@@ -63,6 +66,20 @@ public ActionBeanContext getContext() {
         return context;
     }
 
+    /**
+     * @return the bundle
+     */
+    public ResourceBundle getBundle() {
+        return bundle;

```

Comment:
it the local/bundle is not set it should return a default and not `null` which it does now and causes some testcases to fail.
otherwise bundle should be a required parameter

Reply:
default is set now plus resources relocated to maven compliant location